Makefile.am

```diff
@@ -171,44 +171,32 @@ baseline.info:
 	$(LCOV) -c -i -d $(abs_builddir)/src -o $@
 
 baseline_filtered.info: baseline.info
-	$(abs_builddir)/contrib/filter-lcov.py "/usr/include/" $< $@
+	$(abs_builddir)/contrib/filter-lcov.py -p "/usr/include/" -p "src/leveldb/" -p "src/bench/" -p "src/univalue" -p "src/crypto/ctaes" -p "src/secp256k1" $< $@
```

Comment:
Perhaps abstract out the list of things to filter into a variable, as it's used more than once?